openml-lightgbm-provider/lightgbmlib_build/__commit_id__

```diff
@@ -0,0 +1 @@
+471d0aaaaa8574b9a1da2302e992f61b01e3d155
```

Comment:
can you confirm with Alberto why do we need this file?

as well as, the timestamp and version

Reply:
This is to track the original LightGBM commit from where the lightgbmlib.jar and *.so were built. It's best to leave those three files following "__<name>__" so we can define the build without doubts.

Reply:
I will leave this as it is